start/sw/mobile-app/mobile-installation.md

```diff
@@ -8,20 +8,24 @@ description: Creating mobile applications using Google Android Studio.
 
 # Mobile application
 
-Today wide range of smart mobile devices make it possible to create clever applications. 
+Today wide range of smart mobile devices make it possible to create clever applications.
 
-# Get started
+## Get started
 
-## Install required libraries
+### Install required libraries
 
-`sudo dnf install glibc.i686 glibc-devel.i686 libstdc++.i686 zlib-devel.i686 ncurses-devel.i686`
+```$ sudo dnf install glibc.i686 glibc-devel.i686 libstdc++.i686 zlib-devel.i686 ncurses-devel.i686```
 
-## Install Android Studio
+### Install Android Studio
 
 1. Download [Android Studio](https://developer.android.com/studio/index.html).
 
 2. Unpack the .zip and install to an appropriate location.
 
-3. Navigate to directory where Android Studio was installed and run with `./bin/studio.sh`
+```$ unzip android-studio-ide-${version}-linux.zip```
```

Comment:
Please do not use variables. Some people may not understand them and plainly copied command would not work for them.